tests/lib/rules/no-warning-comments.js

```diff
@@ -50,11 +45,15 @@ ruleTester.run("no-warning-comments", rule, {
         { code: "// any fixme or todo", options: [{ terms: ["fixme", "todo"], location: "anywhere" }], errors: [{ message: "Unexpected 'fixme' comment." }, { message: "Unexpected 'todo' comment." }] },
         { code: "/* any fixme or todo */", options: [{ terms: ["fixme", "todo"], location: "anywhere" }], errors: [{ message: "Unexpected 'fixme' comment." }, { message: "Unexpected 'todo' comment." }] },
         { code: "/* any fixme or todo */", options: [{ location: "anywhere" }], errors: [{ message: "Unexpected 'todo' comment." }, { message: "Unexpected 'fixme' comment." }] },
-        { code: "/* fixme and todo */", errors: [{ message: "Unexpected 'fixme' comment." }] },
+        { code: "/* fixme and todo */", errors: [{ message: "Unexpected 'todo' comment." }, { message: "Unexpected 'fixme' comment." }] },
```

Comment:
This should only be invalid with 2 messages if `{ location: "anywhere" }` is specified as an option.